docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -199,3 +199,14 @@ If you've created a new image that is not rolling compatible, and you've changed
     the image, or the Kubernetes resources that register your domain with the operator.  For example, your servers are caching information from an external database and you've modified the contents of the database.
 
     In these cases, you must manually initiate a restart.
+
+* **Managed Coherence Server safe shutdown**.
+
+    If the domain is configured to use Coherence Clusters, then you will need to increase the Kubernetes graceful timeout value.  
+    When a server is shutdown, Coherence needs time to recover partitions and rebalance the cluster before it is safe to shutdown a second server.
+    Using the Kubernetes graceful termination feature, the Operator will automatically wait until the Coherence HAStatus MBean attribute 
+    indicates that it is safe to shutdown the server.  However, once graceful termination timeout expires, the pod will be deleted regardless. 
```

Comment:
However, once graceful termination timeout expires, -> However, after the graceful termination timeout expires, (once – don’t use it to mean after; use only to mean less than twice)